# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1504515103 -7200
#      Mon Sep 04 10:51:43 2017 +0200
# Node ID d9ef3699b6142dc1b508e27647e6254d0653a5ee
# Parent  21d2114476a48e591751b5e3e399a0dd4791c17d
doc: Improve docstring for repelem().

* scripts/general/repelem.m: @var{} does not allow brackets (), as they are not
valid variable chars.  Rather embed the expression in an @code{} environment
using two separate @var{} environments for the variables.  This suppresses some
Texinfo warnings.

diff --git a/scripts/general/repelem.m b/scripts/general/repelem.m
--- a/scripts/general/repelem.m
+++ b/scripts/general/repelem.m
@@ -24,17 +24,17 @@
 ##
 ## @var{x} must be a scalar, vector, or N-dimensional array.
 ##
 ## A repeat instruction @var{R_j} must either be a scalar or a vector.  If the
 ## instruction is a scalar then each component of @var{x} in dimension @var{j}
 ## is repeated @var{R_j} times.  If the instruction is a vector then it must
 ## have the same number of elements as the corresponding dimension @var{j} of
 ## @var{x}.  In this case, the @var{k}th component of dimension @var{j} is
-## repeated @var{R_j(k)} times.
+## repeated @code{@var{R_j}(@var{k})} times.
 ##
 ## If @var{x} is a scalar or vector then @code{repelem} may be called with just
 ## a single repeat instruction @var{R} and @code{repelem} will return a vector
 ## with the same orientation as the input.
 ##
 ## If @var{x} is a matrix then at least two @var{R_j}s must be specified.
 ##
 ## Note: Using @code{repelem} with a vector @var{x} and a vector for @var{R_j}
